developer_guides/uuid/index.rst

```diff
@@ -0,0 +1,134 @@
+.. _uuid:
+
+UUID Usage in SOF
+#################
+
+Why UUID is needed
+******************
+
+When developing a new audio signal processing component, we
+traditionally implement the component driver with a new unique component
+type declared together, the painpoints with this method including ABI
```

Comment:
not sure what you meant by 'declared together'. together with what?

Reply:
Looks I am making semantic wrong here, I meant that we need to define new component type in the firmware (and shared with the driver) together with the component driver implementation itself.

Reply:
the PR is updated with this part changed, mark it 'resolved'.